src/qtt/algorithms/functions.py

```diff
@@ -480,26 +513,60 @@ def func(params): return cost_gauss_ramsey(x_data, y_data, params, weight_power=
     return fit_parameters, result_dict
 
 
-def plot_gauss_ramsey_fit(x_data, y_data, fit_parameters, fig):
+def plot_gauss_ramsey_fit(x_data, y_data, fit_parameters, fig, initial_parameters=None, show_parameters=False):
     """ Plot Gauss Ramset fit
 
     Args:
-        x_data: Input array with time variable
+        x_data: Input array with time variable (seconds)
         y_data: Input array with signal
         fit_parameters: Result of fit_gauss_ramsey
+        fig: figure number to plot into
+        initial_parameters: if given, also plot function with initial parameters
+        show_parameters: if True, then all parameter values are shown in the plot
     """
-    test_x = np.linspace(0, np.max(x_data), 200)
-    freq_fit = abs(fit_parameters[2] * 1e-6)
+    def get_fit_pars_string(fit_parameters: List[float] = fit_parameters, title: str = "Fit parameters:") -> str:
```

Comment:
We have the convention in qtt to in a single module to either i) annotate all methods and classes with type annotations or ii) don't do any type annotation at all.

We might want to re-evaluate that decision. @QFer @mjwoudstra 
In any case, we need to decide how to annotate things like fit parameters. It would be something like `Union[ List[float], np.ndarray]`, but that is not so pretty.

Reply:
Typehinting a module as a whole is my preference because it's much more efficient if one person typehints a module instead of more persons doing it bit by bit, especially the more difficult types.
Further, it can then be checked by mypy as a whole.

Reply:
I did not want type type annotate the whole module, so let's just remove them from this one function.

Reply:
I did not want type type annotate the whole module, so let's just remove them from this one function.